schemas/djornl/djornl_pheno_assn.yaml

```diff
@@ -0,0 +1,24 @@
+name: djornl_pheno_assn
+type: edge
+delta: false
+
+schema:
+  "$schema": http://json-schema.org/draft-07/schema#
+  title: Phenotype Association Edge
+  description: |
+    GWAS associations produced by analyzing a subset of phenotypes and SNPs
+    in the Arabidopsis 1001 Genomes database. Edge values are significant
+    association scores after FDR correction.
+  type: object
+  required: [assn_score, _from, _to]
+  properties:
+    assn_score:
+      title: Association Score
+      # (float)
+      type: number
```

Comment:
I'm not sure how you have the db set up, but I have been dealing with this edge data as if each edge object has four properties, `_to`, `_from`, `score` (a.k.a. the edge weight), and `type` (the source). It saves having to normalise the names when displaying the data graphically or in tabular form. We can have a mapping of data source to what the edge weight represents. From what the Jacobson team were saying, they are interested in which nodes are connected and in the type of connection (which analysis it came from) between the nodes.

Reply:
It would be useful to have a query that fetches edges filtered by `type` (e.g. `djornl_ppi_liter`, `djornl_pheno_assn`) or by `score`, plus the node data for those edges.

Reply:
Sure, sounds good

Reply:
Pushed an update that combines all the different edge collections into a single one called `djornl_edge` with a `score` and `score_type` field. The score_type field must be one of these strings: [domain_co_occur, gene_coexpr, pheno_assn, ppi_hithru, ppi_liter]